Sources/Overlays/_Testing_WinSDK/Attachments/AttachableAsIWICBitmapSource.swift

```diff
@@ -13,25 +13,11 @@ private import Testing
 public import WinSDK
 
 /// A protocol describing images that can be converted to instances of
-/// ``Testing/Attachment``.
+/// [`Attachment`](https://developer.apple.com/documentation/testing/attachment)
+/// and which can be represented as instances of [`IWICBitmapSource`](https://learn.microsoft.com/en-us/windows/win32/api/wincodec/nn-wincodec-iwicbitmapsource)
+/// by address.
 ///
-/// Instances of types conforming to this protocol do not themselves conform to
-/// ``Testing/Attachable``. Instead, the testing library provides additional
-/// initializers on ``Testing/Attachment`` that take instances of such types and
-/// handle converting them to image data when needed.
-///
-/// You can attach instances of the following system-provided image types to a
-/// test:
-///
-/// | Platform | Supported Types |
-/// |-|-|
-/// | macOS | [`CGImage`](https://developer.apple.com/documentation/coregraphics/cgimage), [`CIImage`](https://developer.apple.com/documentation/coreimage/ciimage), [`NSImage`](https://developer.apple.com/documentation/appkit/nsimage) |
-/// | iOS, watchOS, tvOS, and visionOS | [`CGImage`](https://developer.apple.com/documentation/coregraphics/cgimage), [`CIImage`](https://developer.apple.com/documentation/coreimage/ciimage), [`UIImage`](https://developer.apple.com/documentation/uikit/uiimage) |
-/// | Windows | [`HBITMAP`](https://learn.microsoft.com/en-us/windows/win32/gdi/bitmaps), [`HICON`](https://learn.microsoft.com/en-us/windows/win32/menurc/icons), [`IWICBitmapSource`](https://learn.microsoft.com/en-us/windows/win32/api/wincodec/nn-wincodec-iwicbitmapsource) (including its subclasses declared by Windows Imaging Component) |
-///
-/// You do not generally need to add your own conformances to this protocol. If
-/// you have an image in another format that needs to be attached to a test,
-/// first convert it to an instance of one of the types above.
+/// This protocol is not part of the public interface of the testing library.
```

Comment:
The protocol has a public access control, should line 21 change it to `package protocol`?

Reply:
It must be declared public for technical reasons (needs to be specified in the public protocol conformance on `UnsafePointer`), but it is not API. This is typical for Swift and I wish we had a different keyword for it like `implementationDetailOnly` or something, but it is what it is.